containers.go

```diff
@@ -159,15 +159,14 @@ func (c *Container) MoveVolume(ctx context.Context, params *VirtualMachineMoveDi
 func (c *Container) RRDData(ctx context.Context, timeframe Timeframe, consolidationFunction ConsolidationFunction) (rrddata []*RRDData, err error) {
```

Comment:
this is the problem, we need to add the `...` to make ConsolidationFunction variadic. the point was the the cf is optional so making it variadic means it can be left off in the call. the mashing everything together into a string was the wrong logic